plugins/discourse-details/assets/javascripts/lib/rich-editor-extension.js

```diff
@@ -6,7 +6,6 @@ const extension = {
       attrs: { open: { default: true } },
       content: "summary block+",
       group: "block",
-      draggable: true,
```

Comment:
if `draggable`, it's harder to drag-select a text within any part of the details/summary, so I think removing it is a better default